File: FizzBuzz/Fortran/FizzBuzz.f
      program FizzBuzz
      implicit none
      integer i,n
      write (*, '(a)', advance='no') 'n = '
      read(*,*) n
      if(n<0) then
      	n=100
      end if
      
      write (*,*)
      
      do 100 i=1,n,1
      	if ( mod(i, 15) .eq. 0 ) then
      		write (*,*) 'Fizz Buzz'
      	else if ( mod(i, 5) .eq. 0 ) then
      		write (*,*) 'Buzz'
      	else if ( mod(i, 3) .eq. 0 ) then
      		write (*,*) 'Fizz'
      	else
      		write (*,'(1x, i0)') i
      	end if
  100 continue
      
      write (*,*)
      
      stop
      end program

File: FizzBuzz/CSharp/FizzBuzz/FizzBuzz.cs
﻿using System;

namespace FizzBuzz
{
    public class FB
    {
        public static void Main()
        {
            Console.Write("n = ");
            var line1 = Console.ReadLine().Trim();
            if (int.TryParse(line1, out var n))
                n = (n < 1 ? 100 : n);
            else
                n = 100;
            Console.WriteLine("");
            for (var i = 0; i < n; i++ ) {
                if ((i+1) % 15 == 0) {
                    Console.WriteLine("Fizz Buzz");
                } else if ((i + 1) % 5 == 0) {
                    Console.WriteLine("Buzz");
                } else if ((i + 1) % 3 == 0) {
                    Console.WriteLine("Fizz");
                } else {
                    Console.WriteLine(i+1);
                }
            }
            Console.WriteLine("");
        }
    }
}
